index.html

```diff
@@ -16,11 +17,28 @@
 <div class="font-div">
     <header>
         <div class="header-mobile">
```

Comment:
- [OPTIONAL] I will highly recommend you to consider giving the navbar on the mobile screen a white background. this way it will be a good user experience. because now your logo and menu button on the mobile screen overlays the rest of the elements when the user scrolls so kindly consider fixing it.
your design
![qw](https://user-images.githubusercontent.com/96683843/225096598-88597bdd-7ee4-46e9-ba49-a2df0fcf6d96.jpg)